synapse/models/files.py

```diff
@@ -699,6 +702,44 @@ def getModelDefs(self):
                         'doc': 'The file offset to the beginning of the section'}),
                 )),
 
+                ('file:mime:lnk', {}, (
+                    ('file', ('file:bytes', {}), {
```

Comment:
Is this going to be specific to the environment and not a universal property of the `.lnk` file?

Reply:
Yea, this was a first stab at connecting an LNK file to the actual file. Yea, a file:bytes prop doesn't feel quite right, but neither does hanging something else off of this form.

So I'll remove this and we can have a goback about formalizing a light edge or hanging a secondary prop off the it:exec:proc node?